.github/workflows/regression.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:230225-2136
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c bug_fix_cht_flamelet -s ${{matrix.testscript}}
```

Comment:
makes sense, PR for the test case update?

Reply:
I just opened a pull request for the highlighted test case branch.